features/events/traps/src/test/java/org/opennms/netmgt/trapd/TrapListenerTest.java

```diff
@@ -127,4 +127,16 @@ public static Snmpv3User createUser(String authProtocol,
         user.setSecurityName(securityName);
         return user;
     }
+
+    @Test
+    public void noSubscriberTest() throws Exception {
+        TrapListener listener = new TrapListener(initialConfig);
+        listener.start();
+        Assert.assertEquals(Boolean.FALSE, listener.isRegisteredForTraps());
+        try {
```

Comment:
Try using  `Awaitility` in order to avoid sleeping.

If you can make `60` secs configurable it can be overwritten from the test without much wait.